climada/util/coordinates.py

```diff
@@ -413,9 +416,12 @@ def dist_to_coast_nasa(lat, lon, highres=False):
         os.remove(path_dwn)
         os.chdir(cwd)
 
-    intermediate_shape = None if highres else (3600, 1800)
+    if not highres and intermediate_shape is None:
+        intermediate_shape = (lat.size, lon.size)
```

Comment:
This fallback value doesn't make any sense. It will behave in an undesirable way for almost any possible lat/lon input.

Reply:
You're right, of course. Should have been `np.unique(lat)`, if anything. Doesn't matter now.